src/index.ts

```diff
@@ -332,15 +329,13 @@ const reduceRule = (
       condition?.target &&
       inlineTargets &&
       inlineTargets.some((inlineTarget) =>
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        isSubpath(condition!.target as string, inlineTarget)
+        isSubpath(condition.target as string, inlineTarget),
       )
     ) {
-      const inLineTargetValue = getAttribute(attributes, condition.target);
+      const inLineTargetValue = getAttribute(attributes, condition.target as string);
 
       if (inLineTargetValue) {
-        // @ts-ignore-error
```

Comment:
This was missed because of the `test.only` below, and the ts-ignore-error above.